src/pages/docs/infrastructure/accounts/openid-connect.md

```diff
@@ -22,9 +22,13 @@ These must be exposed with anonymous access on HTTPS. Without this, the OpenID C
 
 The hostname of the URL that these two endpoints are available on must either be configured under **Configuration->Nodes->Server Uri** or set as the first ListenPrefix in the server configuration. 
 
+## Authenticating using OpenID Connect with third party services and tools
+
+If you have a third-party service or tool that supports OpenID Connect, you can add any OIDC account variable into your projects variable set and use the `Octopus.OpenIDConnect.Jwt` variable to get access to the request token that can be used for authenticating. 
```

Comment:
Sorry I was mistaken before, when using the variables it should be `<AccountName>.Octopus.OpenIDConnect.Jwt`